pontoon/base/static/js/translate.js

```diff
@@ -982,17 +982,36 @@ var Pontoon = (function (my) {
       });
     },
 
-
     /*
      * Update progress indicator and value
      */
     updateProgress: function () {
+
+      /*
+       * Count the number of words translated
+       */
+      function wordProgress () {
+        var entities = $("#entitylist .entity .string-wrapper .translation-string");
```

Comment:
In the objects in `Pontoon.entitites` how would I know which of the strings are translated, we only have to count the words from the strings which have already been translated right ?